lib/shared/addon/utils/constants.js

```diff
@@ -238,6 +238,49 @@ var C = {
     ]
   },
 
+  SCHED_NODE_SELECTOR_OPERATOR: [
+    {
+      value: '<',
+      label: 'lt',
```

Comment:
Just set this to a full translation key (`formScheduling.nodeSelector.operator.(label)`) key here and then use the constant instead of computing a similar map in `initOperatorChoices`